tests/unit/amo/components/TestLandingPage.js

```diff
@@ -16,18 +16,15 @@ import I18nProvider from 'core/i18n/Provider';
 import { ErrorHandler } from 'core/errorHandler';
 import { visibleAddonType } from 'core/utils';
 import { dispatchClientMetadata, fakeAddon } from 'tests/unit/amo/helpers';
-import { getFakeI18nInst } from 'tests/unit/helpers';
+import { getFakeI18nInst, createStubErrorHandler } from 'tests/unit/helpers';
```

Comment:
I'll stop commenting but it would be nice to alphabetise all the functions imported. 😄